packaging/package.sh

```diff
@@ -67,22 +75,32 @@ function packaging() {
             exit 2
         fi
     fi
+
     VERSION=$(cd ../; $MVN org.apache.maven.plugins:maven-help-plugin:2.1.1:evaluate -Dexpression=project.version | grep --color=none '^[0-9]\.')
+    BASEVER=$(echo "$VERSION" | sed 's/-SNAPSHOT//g')
+    REALVER=$(echo "$BASEVER" | cut -d '-' -f 1)
+    BRAND=$(echo "$BASEVER" | cut -d '-' -f 2)
```

Comment:
Instead of this, why not pass the branch as a new parameter that is used for the DEFREL option? Or, use sed on the package.sh script to replace the DEFREL option with the branch parameter?